libraries/Microsoft.Bot.Builder/SimplePropertyAccessor.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder
+{
+    public class SimplePropertyAccessor<T> : IPropertyAccessor<T>
+    {
+        private IPropertyContainer propertyContainer;
+        private T defaultValue;
+
+        public SimplePropertyAccessor(IPropertyContainer propertyContainer, string name, T defaultValue = default(T))
+        {
```

Comment:
Null check / throw on all parameters. 

Please add Unit Tests for that as well.